### PR TITLE
Update Github Actions auto_release workflow

### PR DESCRIPTION
**Pull request by carabasdaniel**
Addresses comments on PR: https://github.com/puppetlabs/pdk-templates/pull/393#pullrequestreview-583458528

### CODE REVIEW

**Comment by carabasdaniel** [comment]
Successful workflow run: https://github.com/puppetlabs/puppetlabs-testing/runs/1832174668?check_suite_focus=true
Resulting PR: https://github.com/puppetlabs/puppetlabs-testing/pull/343

**Review by sanfrancrisko — APPROVED** [review]

**Review by david22swan — APPROVED** [review]

File: moduleroot/.github/workflows/auto_release.yml.erb [hunk 1]
```diff
@@ -35,83 +35,71 @@ jobs:
...
         git config --local user.email "action@github.com"
         git config --local user.name "GitHub Action"
         git add .
         git commit -m "Auto-Release prep"
 
     - name: "Get Version"
+      if: ${{ github.repository_owner == '<%= common['owner'] %>' }}
       id: gv
       run: |
         echo "::set-output name=ver::$(cat metadata.json | jq .version)"
```
Comment (DavidS) [inline moduleroot/.github/workflows/auto_release.yml.erb:87]:
```suggestion
        echo "::set-output name=ver::$(cat metadata.json | jq .version | tr -d \")"
```

remove additional quotes from result:

2021-02-05_10MS+0000_444x126

File: moduleroot/.github/workflows/auto_release.yml.erb [hunk 2]
```diff
@@ -35,83 +35,71 @@ jobs:
...
       uses: puppetlabs/peter-evans-create-pull-request@v3
+      if: ${{ github.repository_owner == '<%= common['owner'] %>' }}
       with:
         token: ${{ secrets.GITHUB_TOKEN }}
-        commit-message: "Auto-release prep"
+        commit-message: "Release prep v${{ steps.gv.outputs.ver }}"
         branch: "release-prep"
         delete-branch: true
-        title: "Auto-release prep ${{ steps.gv.outputs.ver }}"
+        title: "Release prep v${{ steps.gv.outputs.ver }}"
```
Comment (DavidS) [inline moduleroot/.github/workflows/auto_release.yml.erb:98]:
```suggestion
        title: "Release prep v${{ steps.gv.outputs.ver }}"
        body: "Automated release-prep through [pdk-templates](https://github.com/puppetlabs/pdk-templates/blob/main/moduleroot/.github/workflows/auto_release.yml.erb)"
```

Avoid advertising for others when we can advertise for our own stuff.

File: moduleroot/.github/workflows/auto_release.yml.erb [hunk 3]
```diff
@@ -35,83 +35,71 @@ jobs:
...
 
     - name: Create Pull Request
       id: cpr
       uses: puppetlabs/peter-evans-create-pull-request@v3
+      if: ${{ github.repository_owner == '<%= common['owner'] %>' }}
       with:
         token: ${{ secrets.GITHUB_TOKEN }}
-        commit-message: "Auto-release prep"
+        commit-message: "Release prep v${{ steps.gv.outputs.ver }}"
         branch: "release-prep"
```
Comment (DavidS) [inline moduleroot/.github/workflows/auto_release.yml.erb:96]:
```suggestion
        branch: "release-prep"
        delete-branch: true
```

clean up afterwards

File: moduleroot/.github/workflows/auto_release.yml.erb (hunk 2 above)
Comment (DavidS) [reply moduleroot/.github/workflows/auto_release.yml.erb:98]: extra points for extracting the generated changelog bits of this release and add them to the text here.

**Review by DavidS — CHANGES_REQUESTED** [review]

**Pull request merged**